enclave_wrangler/objects_api.py

```diff
@@ -536,7 +549,7 @@ def fetch_cset_and_member_objects(
                     'fail-excessive-members', 'comment': f"Failed after {len(cset['member_items'])} members."}])
                 if try_fixing_issues_immediately:
                     call_github_action('resolve-fetch-failures-excess-items')
-        if not cset['member_items'] and cset['expression_items'] and flag_issues:
+        if cset['properties']['isDraft'] or (not cset['member_items'] and cset['expression_items']) and flag_issues:
```

Comment:
## Bug fix: 0-expression drafts were not triggering follow-up
    - Bug fix: If drafts had 0 expression items when first fetched, would not trigger follow-up to update the cset later (mainly to import member items).